media/css/base/mozilla-modal.less

```diff
@@ -70,6 +70,7 @@ html.noscroll body {
                     .image-replaced();
                     border-radius: 50%;
                     border: 2px solid #fff;
+                    transition: transform 0.15s ease;
```

Comment:
I'm of the opinion that we deprecate that `.transition` mixin. All major browsers now support the non-prefixed syntax, it's a visual enhancement... What do you think? File a bug to drop the mixin, or should I use it here?